### ISSUE TITLE
@ingroup not working with concepts

### ISSUE THREAD

**Issue by r-neal-kelly** [doxygen/doxygen#8724]
**Describe the bug**
When using the `@ingroup`, `@{`, and `@}` commands, only one concept is added to the group, even if there are several in the braces.

**Expected behavior**
Any concept in between the braces should be added to the group.

**To Reproduce**
[ingroup concepts.zip](https://github.com/doxygen/doxygen/files/7011106/ingroup.concepts.zip)

**Version**
Windows v1.9.2, using doxywizard.exe

**Additional context**
This is only a minor issue. One can use `@addtogroup` instead of `@ingroup` and that works correctly.

Thank you for Doxygen!

**Comment by albert-github** [comment]
Looks like a regression from 1.9.1 -> 1.9.2

**Comment by doxygen** [comment]
@albert-github Concept support was first added in 1.9.2, so I'm not sure about that.

**Comment by albert-github** [comment]
I just looked at the HTML output, so it might have been giving, by accident, "good" looking information:

**1.9.1**
![image](https://user-images.githubusercontent.com/5223533/130058858-8c192a4d-f681-4591-ac27-da115818e367.png)

**1.9.2**
![image](https://user-images.githubusercontent.com/5223533/130058920-9c3c388d-8253-4db0-acf8-ba0e6b7ce053.png)


With a closer look we now see a heading `Concepts` in 1.9.2 whilst in  1.9.1 it is more a variable, so not really a regression.

**Comment by albert-github** [comment]
When I modify the code a little bit to:
```
/**
* @ingroup concepts_ingroup
*/
template <typename>
concept concept_a = true;
/**
* @ingroup concepts_ingroup
*/
template <typename>
concept concept_b = true;
```

I get:
![image](https://user-images.githubusercontent.com/5223533/130060014-f5a6236d-9892-4092-b2b0-1798a92d2d7c.png)

Which looks OK to me.

The workaround is though a bit contradicting the way it is working for:
```
/**
* @ingroup functions_ingroup
*
* @{
*/
void function_a();
void function_b();
/**
* @}
*/
```

**Comment by doxygen** [comment]
Please verify if the referenced commit fixes the problem.

**Comment by albert-github** [comment]
I just added some documentation to the concepts:
```
/**
* @ingroup concepts_ingroup
*
* @{
*/
/**
 * \brief a concept
 * \details a concept
 */
template <typename>
concept concept_a = true;
/**
 * \brief b concept
 * \details b concept
 */
template <typename>
concept concept_b = true;
/**
* @}
*/
```

and for the concepts I get (though this is independent if the usage of `@ingroup`):
![image](https://user-images.githubusercontent.com/5223533/130069099-9aadd559-e8c3-4d79-b073-74766f20184b.png)

and

![image](https://user-images.githubusercontent.com/5223533/130069193-7ee8f67d-f144-4664-a4f3-c9b9542fda13.png)

Small example just with the ca and b concepts: [example.tar.gz](https://github.com/doxygen/doxygen/files/7014379/example.tar.gz)

**Comment by doxygen** [comment]
@albert-github indeed, now too much information was kept for the next element. I changed it to only transfer the grouping information to the next entry.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.3.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
